core/modules/layout/layout.module

```diff
@@ -1072,7 +1072,7 @@ function layout_get_layout_info($layout_name = NULL) {
       $info[$name] += array(
         'preview' => 'preview.png',
         'stylesheets' => array(
-          'all' => array('layout.css'),
+          'all' => array(str_replace('_', '-', $name)),
```

Comment:
oops;  yeah that makes sense;
On Jan 19, 2015 9:09 PM, "Nathan Haug" <notifications@github.com> wrote:

> In core/modules/layout/layout.module
> <https://github.com/backdrop/backdrop/pull/697#discussion-diff-23199284>:
>
> > @@ -1072,7 +1072,7 @@ function layout_get_layout_info($layout_name = NULL) {
> >        $info[$name] += array(
> >          'preview' => 'preview.png',
> >          'stylesheets' => array(
> > -          'all' => array('layout.css'),
> > +          'all' => array(str_replace('_', '-', $name)),
>
> Ha, and it looks like we need to add .css here. [image: :wink:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/backdrop/backdrop/pull/697/files#r23199284>.
>